code/modules/antagonists/abductor/machinery/dispenser.dm

```diff
@@ -23,7 +25,7 @@
 		amounts[i] = rand(1,5)
 
 /obj/machinery/abductor/gland_dispenser/ui_status(mob/user)
-	if(!isabductor(user) && !isobserver(user))
+	if(!isabductor(user) && !user.mind.has_antag_datum(/datum/antagonist) && !isobserver(user))
```

Comment:
```suggestion
	if(!isabductor(user) && !isobserver(user))
```
Don't specifically check for antagonist status, just checking for the species is fine. If someone becomes an abductor and gets aboard the abductor ship then taking the stuff isn't too bad (They can read the language now after all)